docs/docs/guides/sveltekit.mdx

```diff
@@ -0,0 +1,162 @@
+
+Here, we utilize the `onAuthflowCompleted` custom event, which is triggered upon the successful completion of the auth flow, redirecting the user to a designated route.
+
+```js title="HankoAuth.svelte" showLineNumbers
+<script>
+  import { onMount } from "svelte";
+  import { goto } from "$app/navigation";
+  import { register } from "@teamhanko/hanko-elements";
+  import { env } from "$env/dynamic/public";
+  const hankoApi = env.PUBLIC_HANKO_API_URL;
+
+  const redirectAfterLogin = () => {
+    goto("/dashboard");
+  };
+
+  onMount(async () => {
+    register(hankoApi).catch((error) => {
+      // handle error
+    });
+  });
+</script>
+
+<hanko-auth on:onAuthFlowCompleted={redirectAfterLogin} />
+```
+
+:::info
+When incorporating any Hanko component in your `+page.svelte` file(routes), it is necessary to disable server-side rendering (SSR) by setting `ssr` to false in your `+page.ts` file.
+
+```js filename="routes/login/+page.ts"
+export const ssr = false;
+```
+
+**But why?**
+
+The register function tries to set up the `<hanko-auth>` element for the browser. But, SvelteKit pre-renders pages before displaying them. Since it doesn't have access to the browser's window object during this pre-render phase, the registration of the custom element would fail.
+:::
+
+
+## Add `<hanko-profile>` component
+
+The `<hanko-profile>` component provides an interface, where users can manage their email addresses and passkeys.
+
+```js title="HankoProfile.svelte" showLineNumbers
+<script>
+  import { register } from "@teamhanko/hanko-elements";
+  import { onMount } from "svelte";
+  import { env } from "$env/dynamic/public";
+
+  const hankoApi = env.PUBLIC_HANKO_API_URL;
+
+  onMount(async () => {
+    register(hankoApi).catch((error) => {
+      // handle error
+    });
+  });
+</script>
+
+<hanko-profile />
+```
+
+## Implement logout functionality
+
+You can use `@teamhanko/hanko-elements` to easily manage user logouts. Below, we make a logout button component that you can use anywhere.
+
+```js title="LogoutButton.svelte" showLineNumbers
+<script>
+  import { Hanko } from "@teamhanko/hanko-elements";
+  import { goto } from "$app/navigation";
+  import { env } from "$env/dynamic/public";
+  const hankoApi = env.PUBLIC_HANKO_API_URL;
+
+  const hanko = new Hanko(hankoApi);
+  const logout = () => {
+    hanko.user.logout().catch((error) => {
+      // handle error
+    });
+    goto("/login")
+  };
+</script>
+
+<button on:click={logout}>Logout</button>
+```
+
+## Customize component styles
+
+You can customize the appearance of `hanko-auth` and `hanko-profile` components using CSS variables and parts. Refer to our [customization guide](https://github.com/teamhanko/hanko/tree/main/frontend/elements#ui-customization).
```

Comment:
I had trouble customizing the UI in Sveltekit, if you have more experience in it could you elaborate in the doc.

Reply:
Hey @SimoMay I haven't tried styling in sveltekit. Let me try and get back to you

Reply:
@lfleischmann yeah we can merge this. There will be separate guide for customization

Reply:
Yeah I agree 👍